### ISSUE TITLE
AVT 3 - React Combobox macOS VoiceOver screenreader issues

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#3419]
## Environment
macOS 10.14.5 with VoiceOver
Chrome Version 75.0.3770.100 (Official Build) (64-bit)

## Description
1. Control-Option-Shift-Down Arrow to the main web content
2. Control-Option-Right Arrow. VO announces the Combox title and that it's a text element.
3. Control-Option-Right Arrow. VO announces the optional helper test and that it's a text element.
4. Control-Option-Right Arrow to the ComboBox. VO announces that it's a listbox with 0 items selected. To begin interacting press Control-Option-Shift-Down Arrow.
5. VO announces that you are in a listbox with 0 items selected. Open menu pop-up button 1 of 1, but the menu remains closed.
**Expected Result:** VO should also announce the "Filter..." text
6. Press Space and VO announces "5 results are available use the Up and Down Arrow keys to navigate. Arrow keys move the focus up and down through the list. 
7. Press Space on Option 2 to select it and VO announces, "You are currently on a menu inside a pop-up button. Type text. To display a list of choices press Control-Option-Space." Option 2 is selected and the menu list closes. 
8. Tab to the X icon and VO announces, "You are currently on a button inside of a pop-up button. To click the button press Control-Option-Space.
**Expected Result:** VO should also announce the purpose of the button.
9. Navigate and selected Option 2 again (it was cleared in the last step) and tab away from the Combobox.
10. Tab back to the Combobox and VO announces "Option 2 Option 2 selected."
Note: Did not expect to hear Option 2 announced twice.

**Comment by dakahn** [comment]
Re-tested this on latest Chrome w/Voice Over
_(these issues are with reading focus, not tabbed focus)_

- [ ] "Filter..." placeholder text should be read
- [ ] VO announces the purpose of the Clear Selection button
- [x] Currently selected option is not announced twice

New issues with latest Firefox using JAWS

- [ ] can navigate dropdown using arrows

**Comment by emyarod** [comment]
both reading focus and tabbed focus announce the placeholder text (or initial selected item) and the clear selection button

![Kapture 2020-03-05 at 9 06 19](https://user-images.githubusercontent.com/8265238/75994528-ba446e00-5ec0-11ea-84dc-dd9100e2ba09.gif)

@snidersd @dakahn if you can also confirm this then I believe this issue is resolved

**Comment by emyarod** [comment]
any updates on this based on the recording above?

**Comment by emyarod** [comment]
closing as this issue no longer appears to be present
